charts/templates/crd/apps.kusionstack.io_operationjobs.yaml

```diff
@@ -92,7 +92,7 @@ spec:
           status:
             description: OperationJobStatus defines the observed state of OperationJob
             properties:
-              endTimestamp:
```

Comment:
Is there a user using the old field, and is it necessary to ensure compatibility for this field?

Reply:
currently there is no opjs in clusters